test/integration/test_webrtc_transport.ex

```diff
@@ -167,9 +167,9 @@ defmodule IntegrateTest.WebRtcTransportTest do
     ] = ice_candidates
 
     assert priority1 > priority2
-    assert priority3 > priority2
+    assert priority2 > priority3
     assert priority3 > priority4
-    assert priority5 > priority4
+    assert priority4 > priority5
     assert priority5 > priority6
```

Comment:
It seems that the priority when listen_ips is specified has changed.

Reply:
Since they are specified in [this](https://github.com/versatica/mediasoup/blob/90f7511b7f7dcf8ef562a6be33910654c8b6508a/node/src/Router.ts#L532) order